correios/models/posting.py

```diff
@@ -374,6 +377,20 @@ def freight_package_type(self) -> int:
         """
         return self.freight_package_types[self.package_type]
 
+    @property
+    def non_mechanizable_cost(self) -> Decimal:
+        if self.package_type == Package.TYPE_CYLINDER:
+            return NON_MECHANIZABLE_COST
+        return Package.non_mechanizable_additional_cost(self.width, self.height, self.length)
+
+    @classmethod
```

Comment:
I think this method can not be a `@classmethod` because the package measures need to be [validated](https://github.com/olist/correios/blob/lo-790/correios/models/posting.py#L293) when an instance is created.

What do you think about merge the `non_mechanizable_cost` and `non_mechanizable_additional_cost`. What is the point to be separeted? I think one property is enough.

Reply:
> I think this method can not be a @classmethod because the package measures need to be validated when an instance is created.

Yes he can. If you look closer you will see that this method receives any `width, height, length` and not the `self`.

> What do you think about merge the non_mechanizable_cost and non_mechanizable_additional_cost. What is the point to be separeted? I think one property is enough.

I relied on another codes examples, like `calculate_posting_weight` and `posting_weight` or `calculate_volumetric_weight` and `volumetric_weight`.